src/engine/video/image_base.cpp

```diff
@@ -383,12 +394,14 @@ void ImageMemory::CopyFromImage(BaseTexture *img)
         img_pixels.reserve(img->width * img->height * GetBytesPerPixel());
     }
     catch (std::exception&) {
-        PRINT_ERROR << "Failed to malloc enough memory to copy the image" << std::endl;
+        PRINT_ERROR << "Failed to malloc enough memory to copy the image"
+                    << std::endl;
         return;
     }
 
     for (uint32_t i = 0; i < img->height; ++i) {
-        std::vector<uint8_t>::const_iterator start = _pixels.begin() + i * src_bytes + src_offset;
+        std::vector<uint8_t>::const_iterator start =
+          _pixels.begin() + i * src_bytes + src_offset;
```

Comment:
yep, let's 4 as much as possible, except for alignment as you've done in a few places.